src/frontend/components/InstructorWrapper/index.spec.tsx

```diff
@@ -0,0 +1,71 @@
+import { cleanup, render } from '@testing-library/react';
+import * as React from 'react';
+import { Provider } from 'react-redux';
+
+import { bootstrapStore } from '../../data/bootstrapStore';
+import { appState } from '../../types/AppData';
+import { uploadState } from '../../types/tracks';
+import { InstructorWrapper } from './index';
+
+jest.mock('../InstructorView/index', () => {
```

Comment:
Why do you need to mock this? Can't we just let the real `<InstructorView />` render?

Reply:
I mock it to test if we use it. I don't want to test the `InstructorView` rendering, it should not be done here. If we change the `InstructorView` implementation we should have to change here too and I think it's wrong to do that.

Reply:
I think I'd rather have to change both tests than mock and risk having not-tested stuff, though I can hear your argument too